crates/bevy_math/src/lib.rs

```diff
@@ -19,7 +19,7 @@ mod compass;
 pub mod cubic_splines;
 mod direction;
 mod float_ord;
-pub mod isometry;
+mod isometry;
```

Comment:
Maybe this is super nitpicky, but I feel like this makes the PR a breaking change, right? If so, could you please detail that Isometry2d and Isometry3d are now imported directly from `bevy_math` in the Migration Guide section of the PR description? Great documentation addition btw!

Reply:
The isometry types are completely new types that aren't in any Bevy release yet, so I don't think this counts as a breaking change even though it is "breaking" relative to #14269 which introduced the types